src/jupyterlite_sphinx.py

```diff
@@ -267,7 +270,25 @@ def jupyterlite_build(app: Sphinx, error):
         if app.env.config.jupyterlite_config:
             config = ["--config", app.env.config.jupyterlite_config]
 
+        contents = []
+        if app.env.config.jupyterlite_contents:
+            for content in app.env.config.jupyterlite_contents:
+                contents.extend(["--contents", content])
```

Comment:
```suggestion
        contents = []
        dirs = app.env.config.jupyterlite_contents
        if dirs:
            if isinstance(dirs, str):
                dirs = [dirs]
            for content in dirs:
                contents.extend(["--contents", content])

```
This makes it possible for the value to be a single directory, which I bet is a very common case.